pallets/funding/src/types.rs

```diff
@@ -800,12 +800,31 @@ pub mod inner {
 		pub project_id: ProjectId,
 		pub migration_origins: MigrationOrigins,
 	}
+
+	#[derive(
+		Clone, Copy, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen, Serialize, Deserialize,
+	)]
+	pub enum ParticipationMode {
```

Comment:
I might prefer instead of passing this to the participation extrinsics, to split each participation type into two extrinsics, where the first keeps the multiplier parameter and the otm doesn't. Then you can pass a hardcoded 5 multiplier to the inner function (or pass this ParticipationMode multiplier to the inner function).

Reply:
As mentioned on the other thread, I see your reasoning, but I don't think it adds enough value to justify the effort it would take in refactoring everything. If its just an extrinsic that calls the inner with the same enum, then all we're saving is the conditional weight macro on top of the extrinsic, instead of hardcoding one per extrinsic.